databases/redis/Portfile

```diff
@@ -54,7 +54,7 @@ configure.cppflags-replace \
                     -isystem${prefix}/include
 
 # https://trac.macports.org/ticket/58712
-if {${os.major} <= 18} {
+if {${build_arch} eq "i386"} {
```

Comment:
This will not be correct for universal builds. Use the facilities of the muniversal portgroup to add the `-latomic` flag only for the i386 part of universal builds.